bokehjs/src/lib/models/plots/figure.ts

```diff
@@ -12,22 +12,22 @@ export class FigureView extends PlotView {
   }
 }
 
-export namespace Figure {
+export namespace figure {
```

Comment:
This will still break if api bundle is present. Instead you should add `__view_model__ = "Figure"` in Python and revert those changes.

Reply:
done